src/features/featureGate.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+// Must conform Unleash constants
+type AssistedInstallerFeatureType = 'ASSISTED_INSTALLER_SNO_FEATURE';
```

Comment:
nit: do we need the `ASSISTED_INSTALLER` prefix ? 
can we just call it `SNO_FEATURE` or if we do need that prefix: `AI_SNO_FEATURE`

Reply:
The same constant is added to OCM as well. I would keep the names same to simplify `git grep` by future maintainers.